test/index.js

```diff
@@ -50,15 +50,21 @@ test('read', function (t) {
 
 test('passing data', function (t) {
     // on reads it should be part of the URL
-    var xhr = sync('read', getStub(), {data: {a: 'a', one: 1}});
+    
+    //create a plain object, with no prototypal inheritance, like qs@v3 does 
```

Comment:
No, it won't be required of users. When working with qs#v3.0, my code was breaking, so I figured it would be a good thing to test. But, it's not necessary to have that, I realise now.